src/StaticFAQ/CustomBreadcrumbs.js

```diff
@@ -19,40 +19,73 @@ const style = css`
       margin: 30px 16px;
     }
   }
+  @media only screen and (min-width: 600px) {
+    .breadcrumbs-mobile {
+      display: none;
+    }
+  }
+  @media only screen and (max-width: 600px) {
+    .breadcrumbs-desktop {
+      display: none;
+    }
+  }
 `;
 
 type Props = {|
   breadcrumbs: Array<{ id: ?string, title: string }>,
-  loginToken: string,
+  loginToken: ?string,
 |};
 
-const CustomBreadcrumbs = ({ breadcrumbs, loginToken }: Props) => {
-  const firstCategory = [...breadcrumbs].shift();
-  const lastCategory = [...breadcrumbs].pop();
-  const previousCategory = [...breadcrumbs].slice(-2)[0];
-  const id = previousCategory && previousCategory.id;
-
-  return (
-    <div className="breadcrumbs" data-cy="faq-breadcrumbs">
-      {loginToken && (
-        <span className="desktopOnly">
-          <BackArrow id={id} />
+class CustomBreadcrumbs extends React.Component<Props> {
+  renderBreadCrumbs = (breadcrumbs, maxBreadcrumbsLength) => {
+    const breadcrubsLength = breadcrumbs.map(b => b.title).join().length;
+    if (maxBreadcrumbsLength && breadcrubsLength > maxBreadcrumbsLength) {
+      const length = breadcrumbs.length;
+      breadcrumbs = breadcrumbs.map((b, i) => {
+        if (i !== 0 && i !== length - 1 && i !== length - 2) {
+          return { ...b, title: '...' };
+        }
+        return b;
+      });
+    }
+    return breadcrumbs
+      .slice(1, breadcrumbs.length - 1)
+      .map(breadcrumb => (
+        <Breadcrumb
+          key={breadcrumb.id}
+          breadcrumb={{ id: breadcrumb.id, title: breadcrumb.title }}
+        />
+      ));
+  };
+  render() {
+    const { loginToken, breadcrumbs } = this.props;
+    const firstCategory = [...breadcrumbs].shift();
+    const lastCategory = [...breadcrumbs].pop();
+    const previousCategory = [...breadcrumbs].slice(-2)[0];
+    const id = previousCategory && previousCategory.id;
+
+    const maxBreadcrumbsLengthMobile = 55;
+
+    return (
+      <div className="breadcrumbs" data-cy="faq-breadcrumbs">
+        {loginToken && (
+          <span className="desktopOnly">
+            <BackArrow id={id} />
+          </span>
+        )}
+        <Breadcrumb breadcrumb={{ title: firstCategory.title }} />
+        <span className="breadcrumbs-mobile">
```

Comment:
here and in the L80 you can use `MediaQuery` component from `react-responsive`

Reply:
I could not get jest tests to work with react-responsive so I commented out the second test